### PR TITLE
fix(ui): Calendar icon in task listview

### PR DESCRIPTION
**Pull request by pu-raihan**
Calendar icon of Datetime field in task listview was not rendered properly.
This PR fixes it<br><br>
**Before**
image
**After**
image

### CODE REVIEW

**Comment by pu-raihan** [comment]
#311 is the issue solved here

File: frontend/src/components/ListViews/TasksListView.vue [hunk 1]
```diff
@@ -41,7 +41,9 @@
         <div v-if="column.key === 'due_date'">
           <Tooltip :text="dateFormat(item, 'ddd, MMM D, YYYY | hh:mm a')">
             <div class="flex items-center gap-2 truncate text-base">
-              <CalendarIcon />
+              <div>
+                <CalendarIcon />
+              </div>
```
Comment (shariquerik) [inline frontend/src/components/ListViews/TasksListView.vue:46]:
```suggestion
              <div><CalendarIcon /></div>
```

File: frontend/src/components/ListViews/TasksListView.vue [hunk 2]
```diff
@@ -41,7 +41,9 @@
         <div v-if="column.key === 'due_date'">
           <Tooltip :text="dateFormat(item, 'ddd, MMM D, YYYY | hh:mm a')">
```
Comment (shariquerik) [inline frontend/src/components/ListViews/TasksListView.vue:42]:
```suggestion
          <Tooltip
            :text="item && dateFormat(item, 'ddd, MMM D, YYYY | hh:mm a')"
          >
```

Only show tooltip if date is set

File: frontend/src/components/ListViews/TasksListView.vue [hunk 3]
```diff
@@ -238,7 +240,7 @@ const listBulkActionsRef = ref(null)
 
 defineExpose({
   customListActions: computed(
-    () => listBulkActionsRef.value?.customListActions,
+    () => listBulkActionsRef.value?.customListActions
```
Comment (shariquerik) [inline frontend/src/components/ListViews/TasksListView.vue:243]:
```suggestion
    () => listBulkActionsRef.value?.customListActions,
```

**Review by shariquerik — CHANGES_REQUESTED** [review]

**Comment by pu-raihan** [comment]
@shariquerik I have made the requested changes

**Review by shariquerik — APPROVED** [review]

**Pull request merged**